unstyled_docs/spec/application_service/latest

```diff
@@ -0,0 +1 @@
+r0.1.0.html
```

Comment:
should this not be `latest.html` rather than `latest`?

Reply:
oops, yup. Will fix.